source/WebApp/ts/state/handlers/url/load-gist-async.ts

```diff
@@ -1,15 +1,22 @@
-import { getGistAsync } from '../../../helpers/github/gists.js';
-import languages from '../../../helpers/languages.js';
-import { languageAndTargetMapReverse } from '../helpers/language-and-target-maps.js';
+import type { TargetName } from '../../../helpers/targets';

```

Comment:
TIL

Reply:
Need to add `"importsNotUsedAsValues": "error"` at work.